crates/serpen/tests/fixtures/ast_rewriting/happy_path/models/user.py

```
@@ -0,0 +1,43 @@
+"""
+User model with role enumeration.
+No naming conflicts with other modules.
+"""
+
+from enum import Enum
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)